minichain/p2p.py

```
@@ -44,9 +83,10 @@ async def start(self, port: int = 9000, host: str = "127.0.0.1"):
         self.port = port
         self.host_addr = host
         self.loop = asyncio.get_running_loop()
-        
+
         threading.Thread(target=trio.run, args=(self._trio_main,), daemon=True).start()
         asyncio.create_task(self._asyncio_reader())
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)